webui/src/pages/repositories/repository/pulls/pullDetails.jsx

```diff
@@ -40,7 +40,7 @@ const StatusBadge = ({status}) => {
 };
 
```

Comment:
Still need to fix the changes view when branch is deleted:
![image](https://github.com/user-attachments/assets/38d31c94-9f97-4a19-a9d3-925f821b7c99)

Reply:
Will be handled when handling empty diffs. Same area.